src/storage/v3/vertex_accessor.hpp

```diff
@@ -118,6 +118,9 @@ class VertexAccessor final {
   }
   bool operator!=(const VertexAccessor &other) const noexcept { return !(*this == other); }
 
+  // Dummy function
+  size_t CypherId() const;
```

Comment:
My issue with this is we are not planning to add a global id to vertices, so this is problematic in my opinion. I think it would be better not adding this dummy function at all.